### DISCUSSION TITLE
Bypass init script on startup

### DISCUSSION THREAD

**Discussion by ThomasErvik** [datalust/seq-tickets#2168] [Help]
Is it possible to bypass the init scripts on startup?
Running SEQ in Azure Kubernetes and we have a type'o in a init script. 
Pod crashes ..

**Comment by nblumhardt** [comment]
Hi @ThomasErvik, thanks for getting in touch.

No, there's no way to bypass an init script, but deleting it should have the effect you're after.

Hope this helps, 
Nick
